modules/phase_field/src/postprocessors/GrainTracker.C

```diff
@@ -42,6 +42,8 @@ GrainTracker::GrainTracker(const InputParameters & parameters) :
 {
   if (!_is_elemental && _compute_op_maps)
     mooseError("\"compute_op_maps\" is only supported with \"flood_entity_type = ELEMENTAL\"");
+
+  _empty_2.resize(_n_vars, 0);
```

Comment:
I don't like silent error. Why even return the empty thingie. It seems to me this should mooseError out.